src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -42,33 +45,45 @@ public Command parseCommand(String userInput) throws ParseException {
 
         final String commandWord = matcher.group("commandWord");
         final String arguments = matcher.group("arguments");
+        logger.fine("Command word: " + commandWord);
+        logger.fine("Arguments: " + arguments);
+
         switch (commandWord) {
 
         case AddCommand.COMMAND_WORD:
+            logger.fine("Add command detected");
             return new AddCommandParser().parse(arguments);
 
         case EditCommand.COMMAND_WORD:
+            logger.fine("Edit command detected");
             return new EditCommandParser().parse(arguments);
 
         case DeleteCommand.COMMAND_WORD:
+            logger.fine("Delete command detected");
             return new DeleteCommandParser().parse(arguments);
 
         case ClearCommand.COMMAND_WORD:
+            logger.fine("Clear command detected");
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
+            logger.fine("Find command detected");
             return new FindCommandParser().parse(arguments);
 
         case ListCommand.COMMAND_WORD:
+            logger.fine("List command detected");
             return new ListCommand();
 
         case ExitCommand.COMMAND_WORD:
+            logger.fine("Exit command detected");
             return new ExitCommand();
 
         case HelpCommand.COMMAND_WORD:
+            logger.fine("Help command detected");
             return new HelpCommand();
 
         default:
+            logger.fine("Unknown command detected");
             throw new ParseException(MESSAGE_UNKNOWN_COMMAND);
         }
     }
```

Comment:
Let's remove all config, fine, and finer log messages in other classes. The two in this class should be enough to illustrate their usage.

Reply:
Kept changes for `MainApp` and `AddressBookParser`